.ci/travis/matrix.yml

```diff
@@ -0,0 +1,28 @@
+_performance: &_performance
```

Comment:
Instead of editing the matrix.yml you can just directly edit the travis.yml of this repo: https://github.com/logstash-plugins/logstash-filter-xml/pull/80/files.

You can see that the tests for that PR respect the build matrix from travis.yml of the PR itself: https://app.travis-ci.com/github/logstash-plugins/logstash-filter-xml/builds/254472445

Reply:
Thank's @jsvd for the hint, I fixed that and now the matrix build  [work as expected](https://github.com/logstash-plugins/logstash-filter-xml/pull/78/checks?check_run_id=7873395572).